site/content/api/Command-And-Control-API.md

```diff
@@ -3,38 +3,129 @@ title = "Command & Control API"
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+
+The following property must **NOT** be set for a notification command message (to distinguish it from a command message):
+
+| Name             | Mandatory       | Location                 | Type         |
+| :--------------- | :---------------| :----------------------- | :----------- |
+| *reply-to*       | must NOT be set | *properties*             | *string*     |
+
+
+A notification command message MAY contain arbitrary payload to be sent to the device in an AMQP Data section. The value of the notification command message's *subject* value may provide a hint to the device regarding the format, encoding and semantics of the payload data.
+
+
+
 
 ## Send a Command
 
-### Preconditions
+Business Applications use this operation to send a command to a device.
+Depending on the outcome of sending the command to the AMQP 1.0 network, the application either expects a response from the device 
+(to the response receiver link) or closes the receiver link again.
+
+Note that the outcome is usually determined by the receiving protocol adapter, which is omitted here for simplicity.
 
-1. *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
-2. *Business Application* has established an AMQP link in role *sender* with the target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send messages to and `${tenant_id}` is the ID of the tenant that the device belongs to.
-3. *Business Application* has established an AMQP link in role *receiver* with the source address `control/${tenant_id}/${reply-id}`. Where `${reply-id}` may be any arbitrary string chosen by the client. This link is used by the client to receive the response of the executed command. This link’s source address is also referred to as the reply-to address for the request messages. 
+To send a command to a device, both preconditions need to be fulfilled.
 
-To scope each response to the device the `${reply-id}` could consist of the device-id plus an arbitrary string to scope the command sender instance. If all responses should be received just in the context of the tenant, the device-id could be omitted.   
+The following sequence diagram shows the message flow with a successful outcome:
 
-**Link establishment and Message Flow**
+![Send Command successful outcome](../command_control_Success.png)
 
-The following sequence diagram shows the whole message exchange including the response being sent back to the Business Application.
+The following sequence diagram shows the message flow with an unsuccessful outcome:
 
-![Send Command](../command_control_Success.png)
+![Send Command unsuccessful outcome](../command_control_Unsuccessful_outcome.png)
```

Comment:
Same as above.. I do not think that the application needs to close the links on refejected or released.

Reply:
Agreed.